auth-utils/s3iamcli/s3iamcli/main.py

```diff
@@ -61,6 +62,7 @@ def iam_usage(self):
             [-p <Path>]
         DeleteAccount -n <Account Name>
             [--force]
```

Comment:
Update all APIs that take LDAP credentials accordingly

Reply:
done